Sensory/src/main/java/com/google/android/sensory/HomeFragment.kt

```diff
@@ -107,16 +107,16 @@ class HomeFragment : Fragment() {
       if (linearLayoutUploadStatus.visibility != View.VISIBLE) {
         // may add fade in animation here later
         linearLayoutUploadStatus.visibility = View.VISIBLE
-        updateUploadPercent(0, (syncUploadState as SyncUploadState.Started).totalRequests)
-      } else if (syncUploadState is SyncUploadState.InProgress) {
+        updateUploadPercent(0, (syncUploadState as SyncUploadState.Started).initialSyncRequests)
+      } else if (syncUploadState is SyncUploadState.BatchState) {
         updateUploadPercent(syncUploadState.completedRequests, syncUploadState.totalRequests)
```

Comment:
```suggestion
        updateUploadPercent(syncUploadState.completedRequests, syncUploadState.totalBatchRequests)
```

Is it `totalBatchRequests`? Also, I am unclear on how the UI notification looks like now. Is it something like the progress Bar against starts from 0% for the second time delta upload or will it just readjust the existing progress bar to reflect `totalRequests`
?

Reply:
Changed back to `totalRequests`
Actually now its `currentTotalRequests`